PERSONAL_DATA_DISCLOSURE.md

```diff
@@ -0,0 +1,87 @@
+_Last Updated: Mon, 28 Nov 2022 UTC_
+_Last Reviewed: Mon, 28 Nov 2022 UTC_
+
+## Overview
+
+The purpose of this form is to disclose the types of personal data[^1] (PD) stored by each module.  This information enables those hosting FOLIO to better manage and comply with various privacy laws and restrictions, e.g. GDPR.
+
+It's important to note that PD is not limited to that which can be used to identify a person on it's own (e.g. Social security number), but also data used in conjunction with other data to identify a person (e.g. date of birth + city + gender, or dynamic IP address + date + time), and any information about such a person (e.g. has loaned 5 items, 2 of which are overdue).
+
+For the purposes of this form, "store" includes the following:
+
+* Persisting to storage - Either internal (e.g. Postgres) or external (e.g. S3, etc.) to FOLIO
+* Caching - In-memory, etc.
+* Logging
+* Sending to an external piece of infrastructure such as a queue (e.g. Kafka), database (e.g. Elasticsearch, Library Data Platform), distributed table, etc.
+
+## Personal Data processed by this Module
+
+- [ ] This module does not store any PD.
+- [ ] This module does not process any PD.
+- [ ] This module provides [custom fields](https://github.com/folio-org/folio-custom-fields).
+- [ ] This module stores fields with free-form text (tags, notes, descriptions, etc.)
+- [ ] This module caches PD
+- [ ] This module logs PD
+  - [ ] Log level ERROR includes PD
+  - [ ] Log level WARNING includes PD
+  - [ ] Log level INFO includes PD
+  - [ ] Log level DEBUG includes PD
+- [X] This module transmits PD (including queues, additional databases, etc.)
+
+---
+
+### Identifiable information
+
+- [ ] Passport number / National identification numbers
+- [ ] Driver’s license number
+- [ ] Social security number
+- [ ] Financial account information
+- [X] System Password
+
+### Identifiable information if linked
+
+Information can be combined with others to form a person’s identity.
+
+- [ ] First name
+- [ ] Last name
+- [ ] Gender
+- [ ] Date of birth
```

Comment:
I thought mod-users-bl proxied most of the user records from mod-users.

Does that not include some of these other personal details?